eox_tenant/apps.py

```diff
@@ -49,7 +49,24 @@ class EdunextOpenedxExtensionsTenantConfig(AppConfig):
                         'signal_path': 'celery.signals.task_prerun',
                     },
                 ],
-            }
+            },
+            'cms.djangoapp': {
+                'relative_path': 'signals',
+                'receivers': [
+                    {
+                        'receiver_func_name': 'start_tenant',
+                        'signal_path': 'django.core.signals.request_started',
+                    },
+                    {
+                        'receiver_func_name': 'finish_tenant',
+                        'signal_path': 'django.core.signals.request_finished',
+                    },
+                    {
+                        'receiver_func_name': 'clear_tenant',
```

Comment:
we never use this signal in the lms. Should we add it here?

Reply:
same comment as above